edge-hub/test/Microsoft.Azure.Devices.Edge.Hub.E2E.Test/BackupAndRestoreTest.cs

```diff
@@ -104,7 +104,7 @@ async Task BackupAndRestoreCorruptBackupMetadataTest(ITransportSettings[] transp
             Console.WriteLine("Finished test BackupAndRestoreCorruptBackupMetadataTest");
         }
 
-        [Theory]
+        [Theory(Skip = "Flaky")]
```

Comment:
Tests are disabled to match 1.2 branch:
https://github.com/Azure/iotedge/blob/release/1.2/edge-hub/core/test/Microsoft.Azure.Devices.Edge.Hub.E2E.Test/BackupAndRestoreTest.cs#L107